src/styles/_overrides.scss

```diff
@@ -1,3 +1,25 @@
+@import "colors";
+@import "bootstrap/scss/mixins/buttons";
+
 @import './overrides/swal-overrides';
 @import './overrides/bootstrap-overrides';
 @import './overrides/uppy-overrides';
+
+a {
+  color: $primary-color;
+
+  &:hover {
+    color: darken($primary-color, 7.5%);
+  }
+
+  &.nostyle {
+    text-decoration: inherit !important;
+    color: inherit !important;
+    cursor: auto !important;
+  }
+}
```

Comment:
I've made a global change on every link item to match the primary color of the app. It looks really nice and it's convenient that a link looks different than normal text. If you want to override it and match the original style, use `nostyle` class on a particular link.

Reply:
@Skafec just to mention you here to see this `nostyle` attribute to reset a link style.

Reply:
Great idea 👍